test/send.js

```diff
@@ -702,9 +702,11 @@ describe('send(file).pipe(res)', function () {
         .pipe(res)
       })
 
+      var filePath = path.join(fixtures, 'pets', 'index.html')
+
       request(app)
       .get('/pets/')
-      .expect(200, fs.readFileSync(path.join(fixtures, 'pets', 'index.html'), 'utf8'), done)
+      .expect(200, fs.readFileSync(filePath, 'utf8'), done)
```

Comment:
ESlint was giving me errors of lines too long, this fixed it. Source code lines should never be longer than 80 columns.